sketch.js

```
@@ -0,0 +1,173 @@
+
+var x = document.getElementsByTagName("STYLE");
+var CANVAS_HEIGHT = 1300;
+var CANVAS_WIDTH = 900;
+
+var DATE_WIDTH = 20;
+var DATE_PADDING = 5;
+var DATE_MARGIN = 20;
+
+var Y_SCALE = 2; // for making the y axis bigger
+
+var DATA = [
+  {
+    date: 'Feb 1',
+    target: 18,
+    won: 23,
+  },
+  {
+    date: 'Feb 9',
+    target: 9,
+    won: 9,
+  },
+  {
+    date: 'Feb 20',
+    target: 17,
+    won: 20,
+  },
+  {
+    date: 'Feb 27', 
+    target: 32,
+    won: 39,
+  },
+  {
+    date: 'Mar 1',
+    target: 453,
+    won: 519,
+   },
+   {
+	date: 'Mar 5',
+	target: 57,
+	won: 56,
+   },
+   {
+	date: 'Mar 8',
+	target: 86,
+	won: 95,
+   },
+   {
+	date: 'Mar 12',
+	target: 3,
+	won: 4,
+   },
+   {
+    date: 'Mar 15',
+    target: 365,
+    won: 396,
+   },
+   {
+    date: 'Mar 21',
+    target: 6.5,
+    won: 4,
+   },
+   {
+    date: 'Mar 22',
+    target: 57,
+    won: 55,
+   },
+
+];
+
+function setup() {
+  createCanvas(CANVAS_WIDTH, CANVAS_HEIGHT);
+}
+
+
+function draw() {
+  background(150,20,20);
+  
+  noStroke();
+  fill(255); //target
+  rect(590,160,20,20);
+  text("Target", 620, 175);
+  
+  noStroke();
+  fill(30,30,200); //won
+  rect(590,200,20,20);
+  text("Won", 620, 215);
+  
+  
+  
+  var xOffset = DATE_MARGIN;
+  
+  for (var i = 0; i < DATA.length; i++) {
+    var targetHeight = DATA[i].target * Y_SCALE;
+    var wonHeight = DATA[i].won * Y_SCALE;
+    
+    var check = collidePointRect(mouseX, mouseY, xOffset, CANVAS_HEIGHT - targetHeight, DATE_WIDTH, targetHeight);
```

So you're trying to use `collide.js` here but you're not actually including it anywhere. This should just throw an error saying `collidePointRect` is undefined.